core/discord/src/main/java/app/otakureader/core/discord/DiscordRpcService.kt

```
@@ -0,0 +1,245 @@
+        } else {
+            scope.launch {
+                try {
+                    kizzyRpc?.updatePresence(
+                        Activity(
+                            details = "Idle",
+                            state = "Not reading",
+                            largeImage = DISCORD_LOGO_IMAGE,
+                            largeText = APP_NAME
+                        )
+                    )
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)